Xamarin.Forms.Core/Image.cs

```diff
@@ -19,6 +19,8 @@ public class Image : View, IImageController, IElementConfiguration<Image>
 
 		internal static readonly BindablePropertyKey IsLoadingPropertyKey = BindableProperty.CreateReadOnly("IsLoading", typeof(bool), typeof(Image), default(bool));
 
+		public static readonly BindableProperty TintColorProperty = BindableProperty.Create("TintColor", typeof(Color), typeof(Color), Color.Transparent);
```

Comment:
- use `nameof(TintColor)` instead of `"TintColor"`
- use `Color.Default` or `default(Color)` as default color